### PR TITLE
Stretch dustin hamano

### PR DESCRIPTION
**Pull request by dhamano**
Weird, I pushed stretch before I started on the timer one but it didn't create a pull request.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,73 @@ const siteContent = {
...
+nav01.setAttribute("href","#");
+nav02.setAttribute("href","#");
+
+const navEl = document.querySelector("nav");
+navEl.appendChild(nav01);
+navEl.appendChild(nav02);
+
+const navList = document.querySelectorAll("nav a");
+navList.forEach( (arrayItem, i) => {
+  arrayItem.textContent = siteContent.nav["nav-item-"+(i+1)];
```
Comment (Ian84Be) [inline js/index.js:67]: great way to automate this.

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,73 @@ const siteContent = {
...
+
+function changeH1() {
+  if( ctaH1.textContent === siteContent.cta.h1) {
+    ctaH1.textContent = "BLAH! Is BLAAAAH!";
+  } else {
+    ctaH1.textContent = siteContent.cta.h1;
+  }
+}
+
+ctaBtn.addEventListener("click", changeH1, false)
```
Comment (Ian84Be) [inline js/index.js:112]: excellent.

File: stretch-assignment/digital_timer/index.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  startButton.textContent = "Start 10 second Timer";
+  startButton.setAttribute("id","start");
+  resetButton.setAttribute("id","reset");
+  startButton.setAttribute("onClick","javascript:startTimer()");
+  resetButton.setAttribute("onClick","javascript:resetTimer()");
+  resetButton.textContent = "Reset Timer";
+  document.querySelector(".digits").append(brEl);
+  document.querySelector(".digits").append(startButton);
+  document.querySelector(".digits").append(resetButton);
+})();
```
Comment (Ian84Be) [inline stretch-assignment/digital_timer/index.js:14]: i'm curious to know why you chose to wrap this inside an immediately invoked anonymous function.
this code block would run the same way if it were placed in the global scope.

**Review by Ian84Be — COMMENTED** [review]
excellent job working through this and designing the timer for stretch!

File: stretch-assignment/digital_timer/index.js (hunk 3 above)
Comment (dhamano) [reply stretch-assignment/digital_timer/index.js:14]: Good to know

**Pull request merged**